fission/src/aps/APSDataManagement.ts

```diff
@@ -160,3 +160,27 @@ export async function getFolderData(project: Project, folder: Folder): Promise<D
         return undefined
     }
 }
+
+export async function downloadItem(project: Project, item: Item): Promise<void | undefined> {
```

Comment:
This was leftover from the cleanup from Hunter's branch. Leaving it in here for now. Technically should be a part of the APS integration branch @PepperLola.

Reply:
Moved this section of code into `branp/hunters-download-item` for future use.